tslint.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "gts/tslint.json",
   "rules": {
+    "ban-ts-ignore": false,
```

Comment:
shouldn't be needed with `as any`

Reply:
See https://github.com/gpuweb/cts/pull/25#discussion_r322641297